hackerrank/src/Algorithms/Warmup/CompareTriplets.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Warmup.CompareTriplets;
+
+public class Result
+{
+    public static List<int> compareTriplets(List<int> a, List<int> b)
+    {
+        List<int> result = new List<int>() { 0, 0 };
```

Comment:
```suggestion
        var result = new List<int> { 0, 0 };
        if (a?.Count != b?.Count)
            return result;
```
- 1 Першы радок: MS Learn: [Implicitly-typed local variables](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/declarations#implicitly-typed-local-variables)
- 2 Другі радок: Трэба правяраць сапраўднасць уваходных параметраў. Сэнс праверкі, што даўжыня спісаў павінна быць аднолькавая